cfgmgr/vrfmgr.cpp

```diff
@@ -207,6 +225,60 @@ void VrfMgr::doTask(Consumer &consumer)
         auto vrfName = kfvKey(t);
 
         string op = kfvOp(t);
+        // Mgmt VRF table event handling for in-band management
+        if (consumer.getTableName() == CFG_MGMT_VRF_CONFIG_TABLE_NAME)
+        {
+            SWSS_LOG_DEBUG("Event for mgmt VRF op %s", op.c_str());
+            if (op == SET_COMMAND) 
+            {
+                bool in_band_mgmt_enabled = false;
+                bool mgmt_vrf_enabled = false;
+                for (auto i : kfvFieldsValues(t))
+                {
+                    if (fvField(i) == "mgmtVrfEnabled")
+                    {
+                        if (fvValue(i) == "true")
+                        {
+                            mgmt_vrf_enabled = true;
+                        }
+                        else
+                        {
+                            op = DEL_COMMAND;
```

Comment:
why is this else case required to set `op`. You are already doing this in line 266, right? Please remove both the else cases.

Reply:
ok